src/controller/CommissionerDiscoveryController.h

```diff
@@ -35,6 +35,8 @@
 #include <platform/CHIPDeviceLayer.h>
 #include <protocols/user_directed_commissioning/UserDirectedCommissioning.h>
 
+#if CHIP_DEVICE_CONFIG_ENABLE_COMMISSIONER_DISCOVERY
```

Comment:
How about removing this from the build instead based on the configuration?

Reply:
can you share an example of how to do that?